static/js/components/email/EmailCompositionDialog.js

```diff
@@ -78,6 +79,9 @@ export default class EmailCompositionDialog extends React.Component {
   constructor (props: EmailDialogProps) {
     super(props);
     this.state = editorStateFromProps(props);
+    if (R.isNil(this.state.editorState)) {
```

Comment:
What are we trying to accomplish? I think if `this.state.editorState` is an empty object the `<Editor />` component will instantiate it's own state, which will then be set as the state the first time the `onEditorStateChange` function fires.